WickedEngine/wiHelper.cpp

```diff
@@ -1388,12 +1388,15 @@ namespace wi::helper
 		default:
 		case DebugLevel::Normal:
 			std::cout << str;
+			std::flush(std::cout);
 			break;
 		case DebugLevel::Warning:
 			std::clog << str;
+			std::flush(std::clog);
 			break;
 		case DebugLevel::Error:
 			std::cerr << str;
+			std::flush(std::cerr);
```

Comment:
Just a side note: unlike clog and cout, cerr is unbuffered and implicitly flushed. So strictly speaking, this is not necessary. But it doesn't hurt unless we literally want to get every last bit of performance.